packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -162,47 +162,48 @@ export class TreeItem extends FASTElement {
 
         switch (e.keyCode) {
             case keyCodeArrowLeft:
+                e.preventDefault();
                 this.handleArrowLeft();
                 break;
             case keyCodeArrowRight:
+                e.preventDefault();
                 this.handleArrowRight();
                 break;
             case keyCodeArrowDown:
-                // preventDefault to ensure we don't scroll the page
```

Comment:
Is there a reason we're dropping these comments?

Reply:
Because all the "preventDefault" calls do some variation of that.  Seems redundant.

Reply:
Certain others are not necessary, such as spacebar - https://github.com/microsoft/fast/pull/3951#pullrequestreview-494981192

Reply:
(and even arrow key events could cause side effects other than page scrolling as they bubble, we just picked that one because we saw it)